docs/coding-standards/tests/common-rule/import-paths.md

```
@@ -0,0 +1,23 @@
+# 1. インポートパスのルール
+
+### 規約
```

markdownlint-cli2 (0.20.0)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)